Gemfile.lock

```diff
@@ -1,34 +1,35 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.5.1)
+    better_errors (2.7.1)
       coderay (>= 1.0.0)
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (11.0.1)
-    coderay (1.1.2)
-    commander (4.4.7)
+    byebug (11.1.3)
+    coderay (1.1.3)
```

Comment:
### [coderay](http://coderay.rubychan.de)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.1.2 → 1.1.3

[_~~change-log~~, [source-code](http://github.com/rubychan/coderay)_]





<details>
<summary>Commits</summary>

A change of **56** commits. See the full changes on [the compare page](https://github.com/rubychan/coderay/compare/v1.1.2...v1.1.3).

These are the first 10 commits:
- (161c17d) [port a few tweaks from dsl branch](https://github.com/rubychan/coderay/commit/161c17d2c537a32f38f0dcca75218af69f96102b)
- (d4117f6) [backport .gitignore from dsl branch](https://github.com/rubychan/coderay/commit/d4117f6a90068f3afa1afcc48f7ad9f9d3d3a533)
- (ec89197) [trying to fix tests for Ruby 2.4](https://github.com/rubychan/coderay/commit/ec891978d3756c186104d8d243283f8d3104b85a)
- (e603d98) [test with ruby 2.5, too](https://github.com/rubychan/coderay/commit/e603d988d7723841bc416160c45acefd9f2464eb)
- (d385021) [tweak list of rubies to test](https://github.com/rubychan/coderay/commit/d38502167541a1cd1b505a0e468e0098e3ae7538)
- (913c166) [Remove the statement that is not always reached.](https://github.com/rubychan/coderay/commit/913c1665970ffa4e1da79470fa732aa924569ec0)
- (acf422a) [support for special type 'var'](https://github.com/rubychan/coderay/commit/acf422a444813a84a952b39a569bc0f26c77c5a5)
- (dc767fc) [Add Ruby 2.6 fixing issues](https://github.com/rubychan/coderay/commit/dc767fca8ae78cf5760d3bf1d7e7150fde6c5951)
- (7818585) [Merge pull request #233 from junaruga/hotfix/ruby26-expressi…](https://github.com/rubychan/coderay/commit/7818585e14039e56fb279399ea0a97ee2013068d)
- (baf2b03) [Merge pull request #227 from junaruga/hotfix/not-reached-sta…](https://github.com/rubychan/coderay/commit/baf2b0369b8515d759aca1338f521d8789838ce5)


</details>